1-js/02-first-steps/04-variables/2-declare-variables/solution.md

```diff
@@ -1,21 +1,21 @@
-## The variable for our planet
+## Bizim planetimizin dəyişəni
 
-That's simple:
+Bu asandır:
 
 ```js
-let ourPlanetName = "Earth";
+let ourPlanetName = "Yer";
 ```
 
-Note, we could use a shorter name `planet`, but it might be not obvious what planet it refers to. It's nice to be more verbose. At least until the variable isNotTooLong.
+Nəzərə alın ki, biz `planet` adından istifadə edə bilərdik. Lakin, bu ad ilə hansı planetdən danışdığımız bilinməyəcəkdi. Daha məlumatlı adlar yazmaq yaxşıdır.
```

Comment:
danışdığımız "bilinməyəcəkdir." (the word should end with r)
I think "aydın" better suits in the  sentence than "məlumatlı"